db/migrate/20180719120404_add_role_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddRoleToUsers < ActiveRecord::Migration[5.1]
+  def change
```

Comment:
why do we have two migrations? for the same thing?

Reply:
Deleted.